web/templates/user/_user_show_basedata.html

```diff
@@ -69,7 +69,11 @@
     {% if current_user is privileged_for('user_change') %}
     <div class="col-sm-6 col-md-5 col-lg-4">
         <div class="user"><a class="btn btn-info" href="{{ url_for(".move", user_id=user.id) }}">Umziehen</a></div>
-        <div class="user"><a class="btn btn-info" href="{{ url_for(".suspend", user_id=user.id) }}">Sperren</a></div>
+        {% if lock_unlock == "lock" %}
+          <div class="user"><a class="btn btn-info" href="{{ url_for(".suspend", user_id=user.id) }}">Sperren</a></div>
+        {% else %}
+          <div class="user"><a class="btn btn-info" href="{{ url_for(".end_membership", user_id=user.id, membership_id=lock_unlock) }}">Entsperren</a></div>
+        {% endif %}
```

Comment:
as mentioned above, use `{% if is_locked %}` and call the new endpoint `".unblock"`